### DISCUSSION TITLE
withdraw of coin

### DISCUSSION THREAD

**Discussion by jealalri** [Chia-Network/chia-blockchain#8421] [Support]
I made a withdraw , it was confirm by my wallet, but  never arrived

**Comment by loppefaaret** [comment]
can you provide me with an address from the transaction ?

**Comment by jealalri** [comment]
Fecha
9/20/21, 10:47 PM
Altura
887426
Hash
1af00ac572a09975433d8bec790998390e93814646fd13c6d1937fad0046c4b
I got this in my my space pool
0xd86a7d4ff00ca5545c375093756812d094815db40529738fc9f68c6bf8912eb5, this is my ID in the pool, you can verify, I won and they sent the money to other wallet, they steal me

**Comment by jealalri** [comment]
This is the way you resolve the inquiry, no answer

**Comment by jealalri** [comment]
xch1f0ctuf9v9yj2rjvqmrvrh4d23y3a8mav8gfc6xrex2nhsslx7yss3u3t3p, This was my reward address, where they were suppose to send the token, I already change it

**Comment by BrandtH22** [comment]
@jealalri , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
